taskvine/src/examples/vine_example_blast.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2022- The University of Notre Dame
+# This software is distributed under the GNU General Public License.
+# See the file COPYING for details.
+
+# This example shows some of the data handling features of taskvine.
+# It performs a BLAST search of the "Landmark" model organism database.
+# It works by constructing tasks that download the blast executable
+# and landmark database from NCBI, and then performs a short query.
+
+# The query is provided by a string (but presented to the task as a file.)
+# Both the downloads are automatically unpacked, cached, and shared
+# with all the same tasks on the worker.
+
+# Originally written in C (vine_example_blast.c)
+
+from taskvine import *
+
+import os
+import sys
+import errno
+
+query_string = ">P01013 GENE X PROTEIN (OVALBUMIN-RELATED)\n\
+		QIKDLLVSSSTDLDTTLVLVNAIYFKGMWKTAFNAEDTREMPFHVTKQESKPVQMMCMNNSFNVATLPAE\n\
+		KMKILELPFASGDLSMLVLLPDEVSDLERIEKTINFEKLTEWTNPNTMEKRRVKVYLPQMKIEEKYNLTS\n\
+		VLMALGMTDLFIPSANLTGISSAESLKISQAVHGAFMELSEDGIEMAGSTGVIEDIKHSPESEQFRADHP\n\
+		FLFLIKHNPTNTIVYFGRYWSP\n"
+
+BLAST_URL = "https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/ncbi-blast-2.13.0+-x64-linux.tar.gz"
+
+LANDMARK_URL = "https://ftp.ncbi.nlm.nih.gov/blast/db/landmark.tar.gz"
+
+if __name__ == '__main__':
+    try:
+        m = Manager(port = VINE_DEFAULT_PORT)
+    except IOError as e:
+        print("couldn't create manager:",e.errno)
+        sys.exit(1)
+    print("listening on port",m.port)
+    
+    m.enable_debug_log("manager.log")
+    m.set_scheduler(VINE_SCHEDULE_FILES)
+
+    for i in range(10):
+        t = Task("blastdir/ncbi-blast-2.13.0+/bin/blastp -db landmark -query query.file")
+        
+        t.add_input_buffer(query_string,"query.file", flags=VINE_NOCACHE)
```

Comment:
In these ones lets do:  `cache=False`,  or  `cache=True, flags=VINE_UNPACK`.  We are removing that flags arg in the short future.